examples/factors/main.go

```diff
@@ -31,7 +31,7 @@ var localAddr = flag.String("local_addr", "localhost:9000", "Local address")
 func main() {
 	flag.Parse()
 	ctx := context.Background()
-	if err := weaver.Run(ctx, serve); err != nil {
+	if err := weaver.Run(ctx); err != nil {
```

Comment:
FYI, I really like that we don't pass a lambda here any more. It makes the code much simpler, as the user *has to* know there's a side main implementation that has to get called.